user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx

```diff
@@ -191,16 +191,16 @@ Then, you can use the `getHandler` feature to define a type safe handler of your
 import { getHandler } from '@swarmion/serverless-contracts';
 import { getUserContract } from '@my-project-name/users-contracts';
 
-export const handler = getHandler(getUserContract, async event => {
+export const handler = getHandler(getUserContract)(event => {
```

Comment:
```suggestion
export const handler = getHandler(getUserContract)(async event => {
```

Reply:
We are not `await`ing anything in our functions